### DISCUSSION TITLE
how to disable target/file/line metadata in the output

### DISCUSSION THREAD

**Discussion by morriswinkler** [tokio-rs/tracing#2380] [General]
I use the BunyanFormattingLayer, https://github.com/LukeMathWalker/tracing-bunyan-formatter

```
 Registry::default()
        .with(JsonStorageLayer)
        .with(formatting_layer)
        .init();
    Ok(guards)
  ```
  
  
My output contains 

```
{ "target": "running_rs::command", "line": 71, "file": "src/command.rs" }
```

Is there a way to remove those fields from my output ?

**Comment by davidbarsky** [comment]
This question is better addressed on the tracing-bunyan-formatter repository, but generally speaking, formatting settings like that are typically implemented on the layer that you're using. Looking over its documentation, there doesn't seem to be such an option, so I'd open an issue/PR on tracing-bunyan-formatter.
